ProjectGutenberg/zipf.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import re
+import pickle
+import numpy as np
+import matplotlib.pyplot as plt
+import operator
+
+
+def save_texts(link, filename):
+
+    book_text = requests.get(link).text
+    pickle.dump(book_text, open(filename, "wb"))
+
```

Comment:
It's probably good idea to separate your data fetching functionality into a separate file, so you don't have to run this every time you run an analysis.